test/TritonGPU/pipeline-predicate-assertion-regression.mlir

```diff
@@ -0,0 +1,57 @@
+// RUN: triton-opt %s --tritongpu-pipeline | FileCheck %s
+
+#blocked = #triton_gpu.blocked<{sizePerThread = [1, 4], threadsPerWarp = [4, 8], warpsPerCTA = [4, 1], order = [1, 0], CTAsPerCGA = [1, 1], CTASplitNum = [1, 1], CTAOrder = [0, 1]}>
```

Comment:
can we move this to loop-pipeline.mlir and ideally add a comment and/or a baasic check that the outer loop is not pipelined.